ocean_lib/models/bpool.py

```diff
@@ -585,12 +586,12 @@ def calcPoolInGivenSingleOut(
 
     def get_liquidity_logs(
         self,
-        event_name,
-        from_block,
-        to_block=None,
-        user_address=None,
-        this_pool_only=True,
-    ):
+        event_name: str,
+        from_block: int,
+        to_block: Optional[int] = None,
```

Comment:
`from_block` and `to_block` should be `Optional[BlockIdentifier]`. This happens multiple places in this file.

Reply:
resolvable :)